README.rst

```diff
@@ -20,4 +20,8 @@ human and machine readable way close to the source code. Thanks to
 inheritance and elasticity metadata are organized in the structure
 efficiently, preventing unnecessary duplication.
 
-See the https://tmt.readthedocs.io web for detailed documentation.
+See the `guide <https://tmt.readthedocs.io/en/stable/guide.html>`_
```

Comment:
Just a nitpick, I believe the standard in the repo is to put the links on separate lines. See:

https://raw.githubusercontent.com/teemtee/tmt/b1049bea494eeeeb4b5e59ed5f4aa32d4add8517/docs/questions.rst